app/components/Card/Card.js

```diff
@@ -6,22 +6,21 @@ export const Card = (props) => {
   const { currentCard, cancelCard, cardControl, deckControl } = props
 
   const showAnswer = (answerInfo) => {
-    let $answer = null
-
     if(!deckControl.showDeck){
```

Comment:
Can this be ternarified? It'd probs require extra consts to define the returns, so it wouldn't actually save you any code. It'd just be cool :)